scripts/zones/Beadeaux/mobs/Ge_Dha_Evileye.lua

```diff
@@ -0,0 +1,49 @@
+-----------------------------------
+-- Area: Beadeaux (254)
+--  NM: Ge'Dha Evileye
+-- @spawnmob 17379450
+-- @pos -238 1 -202 (254)
```

Comment:
Would you rather me not add that info? Just do it so when it goes to our server we dont have to ajust it.